### PR TITLE
feature/103final

### PR DESCRIPTION
**Pull request by sharirpcs**
…the DBIOPSRatio in Parameter Labels

### CODE REVIEW

**Comment by robe070** [comment]
The commit description needs to be changed to describe what the commit actually contains. And thats the whole ticket. So, just take the ticket title and put that in the commit description. Probably require another PR.

File: CloudFormationWindows/lansa-stack-type-win.cfn.template [hunk 1]
```diff
@@ -276,9 +276,9 @@
         "StackDifferences": {
             "Metadata": { "Comment1": "Table of differences between the 5 Stack Types" },
             "13DBInstanceClass": { "Development": "db.t3.small", "Test" : "db.t3.large",  "Small" : "db.t3.2xlarge",  "Medium" : "db.r5.xlarge",  "Large" : "db.r5.8xlarge" },
-            "15DBEngine": { "Development": "sqlserver-web", "Test" : "sqlserver-web",  "Small" : "sqlserver-web",  "Medium" : "sqlserver-se",  "Large" : "sqlserver-ee" },
-            "19DBAllocatedStorage": { "Development": "100", "Test" : "100",  "Small" : "500",  "Medium" : "1000",  "Large" : "1000" },
-            "20DBIops": { "Development": "1000", "Test" : "1000",  "Small" : "5000",  "Medium" : "10000",  "Large" : "10000" },
+            "15DBEngine": { "Development": "sqlserver-ex", "Test" : "sqlserver-web",  "Small" : "sqlserver-web",  "Medium" : "sqlserver-se",  "Large" : "sqlserver-ee" },
+            "19DBAllocatedStorage": { "Development": "20", "Test" : "100",  "Small" : "500",  "Medium" : "1000",  "Large" : "1000" },
+            "DBIOPSRatio": { "Development": "0", "Test" : "1000",  "Small" : "5000",  "Medium" : "10000",  "Large" : "10000" },
```
Comment (robe070) [inline CloudFormationWindows/lansa-stack-type-win.cfn.template]: Only the Development value is correct. All the others are meant to be the default value for DBIOPS which is 10.

Comment (sharirpcs) [reply CloudFormationWindows/lansa-stack-type-win.cfn.template]: I have set the default value to 10 now

**Comment by sharirpcs** [comment]
I have added the ticket title in the commit message  and squashed the previous commits.Please review it

**Comment by robe070** [comment]
All looks good - just need CI checks to complete.

**Pull request merged**